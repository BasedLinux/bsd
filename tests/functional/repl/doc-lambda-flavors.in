:l doc-comments.bsd
:doc nonStrict
:doc strict
:doc strictPre
:doc strictPost
