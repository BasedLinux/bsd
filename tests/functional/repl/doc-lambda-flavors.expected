Bsd <bsd version>
Type :? for help.

bsd-repl> :l doc-comments.bsd
Added <number omitted> variables.

bsd-repl> :doc nonStrict
Function `nonStrict`\
  … defined at /path/to/tests/functional/repl/doc-comments.bsd:60:5


My syntax is not strict, but I'm strict anyway.

bsd-repl> :doc strict
Function `strict`\
  … defined at /path/to/tests/functional/repl/doc-comments.bsd:65:5


I don't have to be strict, but I am anyway.

bsd-repl> :doc strictPre
Function `strictPre`\
  … defined at /path/to/tests/functional/repl/doc-comments.bsd:71:5


Here's one way to do this

bsd-repl> :doc strictPost
Function `strictPost`\
  … defined at /path/to/tests/functional/repl/doc-comments.bsd:76:5


Here's another way to do this
